<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>926</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="SidebarLayout">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="PlayerAPhoto">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>[玩家头像]</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PlayerAName">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="PieceLabel">
            <property name="text">
             <string>现在是黑方下棋</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="PieceMode">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="LeftTimeLabel">
            <property name="text">
             <string>剩余思考时间</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LeftTime">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Rounded MT Bold</family>
              <pointsize>48</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="PlayerBPhoto">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string>[玩家头像]</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="PlayerBName">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>BAKA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="StartMenu">
    <property name="title">
     <string>开始(&amp;E)</string>
    </property>
    <addaction name="NewgameAction"/>
    <addaction name="saveChessAction"/>
   </widget>
   <widget class="QMenu" name="PlayerMenu">
    <property name="title">
     <string>玩家(&amp;P)</string>
    </property>
    <addaction name="playerOptionAction"/>
   </widget>
   <widget class="QMenu" name="OptionMenu">
    <property name="title">
     <string>设置(&amp;O)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="optionAction"/>
   </widget>
   <widget class="QMenu" name="AboutMenu">
    <property name="title">
     <string>关于(&amp;I)</string>
    </property>
    <addaction name="playingMethodAction"/>
    <addaction name="aboutGames"/>
   </widget>
   <addaction name="StartMenu"/>
   <addaction name="PlayerMenu"/>
   <addaction name="OptionMenu"/>
   <addaction name="AboutMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="NewgameAction">
   <property name="text">
    <string>新游戏(&amp;N)</string>
   </property>
  </action>
  <action name="saveChessAction">
   <property name="text">
    <string>保存棋盘(未实现)(S)</string>
   </property>
  </action>
  <action name="playerOptionAction">
   <property name="text">
    <string>设置(&amp;A)</string>
   </property>
  </action>
  <action name="playerBAction">
   <property name="text">
    <string>玩家2(&amp;B)</string>
   </property>
  </action>
  <action name="playingMethodAction">
   <property name="text">
    <string>玩法(&amp;U)</string>
   </property>
  </action>
  <action name="aboutGames">
   <property name="text">
    <string>软件相关(&amp;M)</string>
   </property>
  </action>
  <action name="optionAction">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
